@vates/node-vsphere-soap/README.md

```diff
@@ -24,7 +24,7 @@ $ npm install node-vsphere-soap --save
 
 ## Sample Code
 
-### To connect to a vCenter server:
```

Comment:
Generally headings shouldn't end in punctuation, that's reserved for prose. (I haven't changed headings that end in `?`, but they're a special case.)